edx/frontend-app-library-authoring
# [BD-14] List filtering and library details

#### Description

This introduces:

* Fixes for the remaining Paragon upgrade issues
* A fix for assets not showing immediately after being uploaded
* A header account dropdown menu that better matches Studio design
* Library list filtering
* Editing of library details

#### Manual testing

1. Follow the manual testing instructions in https://github.com/edx/frontend-app-library-authoring/pull/4 until step 16, so that you have a v2 library to work with.  Use this PR's branch (`arbrandes/filtering-and-details`), though.

1. If https://github.com/edx/edx-platform/pull/24510 hasn't been merged when testing this, check that PR out and restart Studio.  Otherwise, filtering won't work.

1. Make sure the `ORGANIZATIONS_APP` feature flag is set to `true` for Studio.  In the latest devstack, that is the default.  If unset, the org filter dropdown won't work.

1. In the library page of the library you just created, click on Settings > Details.

    <details>
    <summary>Screenshot</summary>

    ![Screenshot_2020-08-22_22-55-32](https://user-images.githubusercontent.com/759355/90969094-c81b9900-e4ca-11ea-88e3-2950fc9aacde.png)
    </details>

1. Edit it as you see fit, then press submit.

    <details>
    <summary>Screenshot</summary>

    ![Screenshot_2020-08-22_23-02-59](https://user-images.githubusercontent.com/759355/90969148-b686c100-e4cb-11ea-845e-8e1fffa31694.png)
    </details>

1. You will be taken back to the library page.  If you changed the title, you should see the new one here.

1. Go back to the library list, and create a few libraries.  Also create a second organization, and put some libraries in there.  Finally, use the filter fields on the right and check that the right libraries are being shown.

    <details>
    <summary>Screenshot</summary>

    ![Screenshot_2020-08-22_23-10-20](https://user-images.githubusercontent.com/759355/90969259-d8347800-e4cc-11ea-8bb6-7aaf47e35fd1.png)
    </details>
